cmd/tk/util.go

```diff
@@ -56,3 +58,22 @@ func writeJSON(i interface{}, path string) error {
 
 	return nil
 }
+
+// writeJSON writes the given object to the path as a JSON file
+func writeJsonnet(i interface{}, path string) error {
+	out, err := json.MarshalIndent(i, "", "  ")
+	if err != nil {
+		return fmt.Errorf("marshalling: %s", err)
+	}
+
+	main, err := formatter.Format(path, string(out), formatter.DefaultOptions())
+	if err != nil {
+		return fmt.Errorf("writing %s: %s", path, err)
```

Comment:
`formatter.Format` does not write the file, I have added a suggestion to update the error message:

```suggestion
		return fmt.Errorf("formatting %s: %s", path, err)
```